### ISSUE TITLE
Some Improvements?

### ISSUE THREAD

**Issue by dufferzafar** [Distroshare/distroshare-ubuntu-imager#2]
Hey, I finally got to testing the created ISO in a virtual machine and it worked! Thanks for the great work.

I have a few minor suggestions in mind:
- A way to backup config files and settings. Since we're backing all additional softwares installed and all, it only makes sense that we backup config too. One way might be where we allow the user to just pause the script after copying the file system and the user is then free to add config or other stuff that he wants. After which the user 'presses any key to continue' and the script resumes.
- Display progress during the "Copy the filesystem" step. I think `rsync` has a method of displaying progress. [Link](http://serverfault.com/questions/219013/showing-total-progress-in-rsync-is-it-possible/441724#441724)
- Make uninstall of additional software optional, as those packages will be needed again when the script is run. I think we could just add some command line params to the script so a user can decide whatever they want.

**Comment by parthlawate** [comment]
Hi, 

When you mean config and settings, i assume you mean the .config folder in the home directory ?

**Comment by hugegreenbug** [comment]
Good suggestions. Yeah, I am also not sure what you mean by config and settings. If you do mean the .config folder, then you can put that in /etc/skel and all users will have those settings.

**Comment by dufferzafar** [comment]
Yes, I meant the .config folder (along with dotfiles present in the home directory.)

Stuff like, `~/.zshrc` etc
